ctapipe/containers.py

```diff
@@ -619,6 +634,10 @@ class SimulatedCameraContainer(Container):
         description="true impact parameter",
     )
 
+    service = Field(
```

Comment:
This should not be in the `SimulatedShowerContainer` because "service" information does not change per shower (anything under shower should be updated per shower).  Instead, it should be under `SimulationConfigContainer` I think (which in any case needs some re-structuring).

Reply:
At least that is if we consider this to be Configuration information, and not simulation service information.   If we really want it as Service info, we should start a new top-level container for that.   But I think this is really part of the input configuration of the simulations, so having it under `source.simulation_config` is fine. Where it goes in tthe data file is another option: it could be in either `/configuration/simulation/`  or `/simulation/service/`

Reply:
But source.simulation_config has no attribute "tel[tel_id]" and this data changes per telescope

Reply:
@clara-escanuela you can add it :)

Reply:
This would mean adding a field like this to the `SimulationConfigurationContainer`:

```
tel = Field(default_factory=partial(Map, TelescopeSimulationConfigurationContainer))
```

and defining the `TelescopeConfigurationContainer`. for now this can be what you now call "PixelMonitoring" or it can have that as a child field, I don't have a strong opininon here.